lib/vssh/libssh2.c

```diff
@@ -583,7 +587,8 @@ static CURLcode ssh_knownhost(struct connectdata *conn)
         if(addrc)
           infof(data, "Warning adding the known host %s failed!\n",
                 conn->host.name);
-        else if(rc == CURLKHSTAT_FINE_ADD_TO_FILE) {
+        else if(rc == CURLKHSTAT_FINE_ADD_TO_FILE || 
+          rc == CURLKHSTAT_FINE_REPLACE) {
```

Comment:
wrong indent-level (but we can fix that when we merge)

Reply:
let me know if the new ident is better ?